packages/components/src/components/step-group/step-group.stories.ts

```diff
@@ -44,240 +53,184 @@ export default {
+          <p slot="label">Label</p>
         </sd-step>
 
-        <sd-step size="lg" orientation="horizontal">
-          <p slot="label">Reprehenderit qui in e name</p>
+        <sd-step size="lg" orientation="horizontal" state="disabled">
+          <p slot="label">Label</p>
         </sd-step>
       </sd-step-group>
 
-      <sd-button class="w-20 mt-8" size="sm" id="previous">Previous</sd-button>
-      <sd-button class="w-20 mt-8" size="sm" id="next">Next</sd-button>
-
-      <script type="module">
-        const stepGroup = await document.querySelector('sd-step-group#set-active');
+      <sd-step-group size="sm" orientation="horizontal" active-step="1">
+        <sd-step size="sm" orientation="horizontal" state="default">
+          <p slot="label">Label</p>
+        </sd-step>
 
-        const nextBtn = document.querySelector('sd-button#next');
-        const prevBtn = document.querySelector('sd-button#previous');
+        <sd-step size="sm" orientation="horizontal" state="current">
+          <p slot="label">Label</p>
+        </sd-step>
 
-        nextBtn.addEventListener('click', () => {
-          stepGroup.setActiveStep(stepGroup.activeStep + 1);
-        });
+        <sd-step size="sm" orientation="horizontal" state="disabled">
+          <p slot="label">Label</p>
+        </sd-step>
+      </sd-step-group>
 
-        prevBtn.addEventListener('click', () => {
-          stepGroup.setActiveStep(stepGroup.activeStep - 1);
-        });
-      </script>
-    `;
-  }
+      <div class="flex gap-12 h-[20vh]">
+        <sd-step-group size="lg" orientation="vertical" active-step="1">
+          <sd-step size="lg" orientation="vertical" state="default">
+            <p slot="label">Label</p>
+          </sd-step>
+
+          <sd-step size="lg" orientation="vertical" state="current">
+            <p slot="label">Label</p>
+          </sd-step>
+
+          <sd-step size="lg" orientation="vertical" state="disabled">
+            <p slot="label">Label</p>
+          </sd-step>
+        </sd-step-group>
+
+        <sd-step-group size="sm" orientation="vertical" active-step="1">
+          <sd-step size="sm" orientation="vertical" state="default">
+            <p slot="label">Label</p>
+          </sd-step>
+
+          <sd-step size="sm" orientation="vertical" state="current">
+            <p slot="label">Label</p>
+          </sd-step>
+
+          <sd-step size="sm" orientation="vertical" state="disabled">
+            <p slot="label">Label</p>
+          </sd-step>
+        </sd-step-group>
+      </div>
+    </div>
+  `
 };
 
 /**
- * This sample shows how to use the not-interactive attribute with content icons. The border-radius of an sd-step is larger when the not-interactive attribute is set to `true`.
- * The size of the icon is recommended to be 48px for the 'lg' size and 32px for the 'sm' size. The size can be set using font-size.
+ * Use the `active-step` attribute to set the current step in the step-group.
```

Comment:
* Use the `active-step` attribute to set the current step.

Reply:
Show:

![image](https://github.com/user-attachments/assets/8a02d1bf-094e-4d8d-8d56-5ae8f120a289)